### PR TITLE
Switch from Travis to Github Actions

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: .github/workflows/build-check.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  docker-example:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Gen Go
+      run: make gen-go
+    - name: Gen Java
+      run: make gen-java
+    - name: Gen Python
+      run: make gen-python gen-swagger
```
Comment (arminru) [inline .github/workflows/build-check.yaml:18]:
```suggestion
      run: make gen-python
```

Comment (bogdandrutu) [reply .github/workflows/build-check.yaml]: done.

File: .github/workflows/build-check.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+on:
+  pull_request:
+    branches: [ master ]
+#    paths:
+#      - '**.proto'
+
+jobs:
+  docker-example:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (yurishkuro) [inline .github/workflows/build-check.yaml:11]: "steps" means they run sequentially, right? would it make more sense to code them as "jobs" to run concurrently?

File: .github/workflows/build-check.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Build Check
+on:
+  pull_request:
+    branches: [ master ]
+#    paths:
+#      - '**.proto'
+
+jobs:
+  docker-example:
```
Comment (arminru) [inline .github/workflows/build-check.yaml:9]:
```suggestion
  build:
```
or something like that

Comment (bogdandrutu) [reply .github/workflows/build-check.yaml:11]: Done.

File: .github/workflows/build-check.yaml (hunk 2 above)
Comment (bogdandrutu) [reply .github/workflows/build-check.yaml:11]: Done.

**Comment by bogdandrutu** [comment]
Recreated the PR in #244 to trigger the actions.

**Pull request closed without merging**